### DISCUSSION TITLE
Has the service been converted to a paid service?

### DISCUSSION THREAD

**Discussion by Tanggue** [aclap-dev/video-downloadhelper#185] [Help and Q&A]
### Your question

After the latest update, the following message appears.
“Without premium status, HLS downloads can only be performed 120 minutes after the previous download.”
-
Previously, unlimited downloads were available for free, but with the recent update, has the DownloadHelper service become a paid service?

### Addon & CoApp version and details

version: 8.2.1.1

**Comment by paulrouget** [comment]
Normally, nothing has changed.

What OS, what browser?

**Reply by Tanggue** [reply]
The OS is Windows.
I've used both Chrome and Firefox,
In Chrome, a message will appear telling you that you can download one file every 120 minutes unless you switch to premium.
In FireFox, the message "EPERM: operation not permitted, mkdir 'D:\" appears.

In the case of Chrome, before the update, you could download any number of videos indefinitely, but suddenly you had to pay premium to download them.
In the case of FireFox, they said it would improve with the update, but the same message still appears.

**Reply by Tanggue** [reply]
After a message appeared telling me that I had to pay for premium to download it, I was moved to that site.
![2024-02-21 133341](https://github.com/aclap-dev/video-downloadhelper/assets/160303209/b39c90a1-eb6d-4a67-a02a-084f53de07df)

**Reply by paulrouget** [reply]
> In the case of Chrome, before the update, you could download any number of videos indefinitely:

If that was the case, that's normal behavior. HLS download is only available for Firefox.

> In the case of FireFox, they said it would improve with the update, but the same message still appears.

Still waiting for Mozilla to push the 8.2.1 update.

**Reply by Tanggue** [reply]
Thank you for answer. I didn't know that only FireFox had free unlimited downloads.
So, is it possible to change the settings differently in the future to have unlimited free downloads on Chrome?

**Reply by paulrouget** [reply]
No, sorry.

**Comment by Aziiiz1992** [comment]
I also have a question, why can't I download more than 60 minutes? knows that someone what you have to set for it. I have already bought the license as well

**Reply by GrampaWildWilly** [reply]
You just opened discussion #1258.  Do not spread you question to multiple locations.  That won't help you.  Confine your posts to that discussion.  You will get help there.  Keep discussions on a single topic.  Don't broadcast your question to discussions of entirely different topics.
